client/src/commonMain/kotlin/com/algolia/search/model/indexing/Partial.kt

```diff
@@ -167,9 +167,13 @@ public sealed class Partial {
             val json = buildJsonObject {
                 put(
                     value.attribute.raw,
-                    buildJsonObject {
-                        key?.let { put(Key_Operation, key) }
-                        put(KeyValue, value.value)
+                    key?.let {
+                        buildJsonObject {
+                            put(Key_Operation, key)
+                            put(KeyValue, value.value)
+                        }
+                    } ?: run {
+                        value.value
                     }
```

Comment:
```suggestion
                    } ?: value.value
```
_nitpick: I believe `run` is unnecessary here :)_